install.sh

```diff
@@ -75,14 +75,20 @@ install_nvm_from_git() {
       exit 1
     }
   else
+    echo "=> Cleanup old nvm dir '$INSTALL_DIR'"
+    command rm -rf "$INSTALL_DIR/nvm-exec" "$INSTALL_DIR/nvm.sh"
     # Cloning to $INSTALL_DIR
     echo "=> Downloading nvm from git to '$INSTALL_DIR'"
     command printf "\r=> "
     mkdir -p "$INSTALL_DIR"
-    command git clone "$(nvm_source)" "$INSTALL_DIR" || {
+    command git init "$INSTALL_DIR"
+    command git --git-dir="$INSTALL_DIR"/.git remote add origin "$(nvm_source)"
+    command git --git-dir="$INSTALL_DIR"/.git fetch --tags
```

Comment:
let's do `git fetch origin --tags` here, in case someone (like me) has multiple sources set up in their global git config.